geps/gep-1686.md

```diff
@@ -1,7 +1,7 @@
 # GEP-1686: Mesh conformance testing plan
 
 - Issue: [#1686](https://github.com/kubernetes-sigs/gateway-api/issues/1686)
-- Status: Provisional
+- Status: Implementable
```

Comment:
At the same time, it would be good to merge #1957 which fixes a couple of issues with running the initial "Mesh" conformance tests.

Reply:
That change mostly LGTM and should merge soon but don't think that needs to block this change.